lib/Conversion/TosaToEmitC/TosaToEmitC.cpp

```diff
@@ -881,6 +881,8 @@ void populateTosaToEmitcPatterns(MLIRContext *ctx,
                                                            "emitc::tosa::add");
   patterns.add<ArithmeticRightShiftOpConversion>(
       ctx, "emitc::tosa::arithmetic_right_shift");
+  patterns.add<CallOpBroadcastableConversion<tosa::GreaterEqualOp>>(
```

Comment:
Can we have this below the Equal case? (Keeping these a bit more sorted will make it easier).